lib/src/main/kotlin/com/swmansion/starknet/data/types/transactions/TransactionPayload.kt

```diff
@@ -31,8 +32,17 @@ data class DeployTransactionPayload(
 
 data class DeclareTransactionPayload(
     val contractDefinition: ContractDefinition,
+    val senderAddress: Felt,
     val maxFee: Felt,
     val nonce: Felt,
     val signature: Signature,
     val version: Felt,
-)
+) {
+    constructor(
```

Comment:
Why is this needed?

Reply:
I guess it's only used in tests using old declare transactions (unsigned). I'll remove this as we don't really want to support tx version 0.